mathics/core/characters.py

```diff
@@ -510,7 +510,8 @@
     'ImaginaryI': '\uF74E',
     'ImaginaryJ': '\uF74F',
     'ImplicitPlus': '\uF39E',
-    'Implies': '\uF523',
+# Implies' unicode in Mathematica is '\uF523', but it will show nothing without the specific font.
+    'Implies': '\u21D2',
```

Comment:
The problem with the change here is that it breaks `a \[Implies] b` (we should add a test).